ios/browser/api/features/features.h

```diff
@@ -78,6 +78,8 @@ OBJC_EXPORT
 @property(class, nonatomic, readonly) Feature* kBraveHttpsByDefault;
 @property(class, nonatomic, readonly) Feature* kBlockAllCookiesToggle;
 @property(class, nonatomic, readonly) Feature* kModernBrowserMenuEnabled;
+@property(class, nonatomic, readonly) Feature* kBraveTranslateFeature;
+@property(class, nonatomic, readonly) Feature* kBraveAppleTranslateFeature;
```

Comment:
Name's are wrong, should match the features (`kBraveTranslateEnabled`, `kBraveAppleTranslateEnabled`)